### ISSUE TITLE
The first 3 and then several other links are 404

### ISSUE THREAD

**Issue by szabgab** [NicholasLYang/rustjobs.fyi#11]
I just bumped into https://www.rustjobs.fyi/ and found that several (many?) links return page not found.

**Comment by azzamsa** [comment]
Yes, it is not maintained anymore. Use [r/rust who is hiring](https://new.reddit.com/r/rust/?f=flair_name%3A%22%F0%9F%92%BC%20jobs%20megathread%22) instead.

**Comment by szabgab** [comment]
Thanks for the clarification!

**Comment by azzamsa** [comment]
I am not the maintainer, but the listing is outdated. I use the link I mentioned above.
